### PR TITLE
Skip metrics tests on Windows.

### PR DESCRIPTION
**Pull request by janisz**
## High-level description

Skip metrics tests that requires dcos_metrics output configured on telegraph since is not supported on Windows.

See:

* https://github.com/dcos/telegraf/blame/d3666e/plugins/outputs/dcos_metrics/README.md#L5
* https://jira.d2iq.com/browse/D2IQ-65777
* https://jira.d2iq.com/browse/D2IQ-65401

## Corresponding DC/OS tickets (required)

  - [D2IQ-65708](https://jira.d2iq.com/browse/D2IQ-65708) 

## Related tickets (optional)

  - [D2IQ-ID](https://jira.mesosphere.com/browse/D2IQ-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @jongiddy @jeschkies @DominikDary

**Comment by janisz** [comment]
@mesosphere-mergebot changelog-not-required test change

**Comment by janisz** [comment]
:wave: @jeschkies @DominikDary  Can you review, please?

File: packages/dcos-integration-test/extra/test_metrics.py [hunk 1]
```diff
@@ -696,10 +700,14 @@ def check_executor_metrics_metadata():
...
 
 
-@pytest.mark.supportedwindows
 def test_metrics_node(dcos_api_session):
     """Test that the '/system/v1/metrics/v0/node' endpoint returns the expected
     metrics and metric metadata.
+
+    This test is not supported on Windows since dcos_metrics
+    plugin is not supported on Windows
+    https://github.com/dcos/telegraf/blame/d3666e/plugins/outputs/dcos_metrics/README.md#L5
```
Comment (DominikDary) [inline packages/dcos-integration-test/extra/test_metrics.py]: we mave metrics on windows, see here https://docs.google.com/document/d/1D7K6X8CDIiLkl66f6NqFMDZLGgoBU4QQsB2eT8Psonk/edit#heading=h.ev9f07y3dpze

**Comment by janisz** [comment]
@mesosphere-mergebot sync

**Pull request merged**